### PR TITLE
refactor(opacity checkerboard)!: refactor from component into commons

### PR DESCRIPTION
**Pull request by jenndiaz**
## Description

BREAKING CHANGE - would require removing a component 

Refactors Opacity Checkerboard from a component to be with in `components/commons/opacity-checkerboard.css` this addresses concerns around inclusion order overwriting css values (#2139) and simplifies opacity checkerboard. 

**Alternative solution** to the GitHub issue: move css unnecessary to implement the background pattern, specifically `inline-size`, ` block-size` from the `index.css` and add to inline styling for the component docs site and storybook. 

## How and where has this been tested?

Please tag yourself on the tests you've marked complete to confirm the tests have been run by someone other than the author.

### Validation steps

**Docs Site**
Opacity Checkerboard should appear correctly on: 
- [ ] colorhandle 
- [ ] mods with in opacity checkerboard still work as expected (look at `--mod-opacity-checkerboard-position` with in colorhandle) 
- [ ] colorslider 
- [ ] thumbnail
- [ ] swatch 

**Storybook**
Opacity Checkerboard should appear correctly on: 
- [ ] colorhandle 
- [ ] colorslider 
- [ ] thumbnail
- [ ] swatch 

### Regression testing

Validate:

1. The documentation pages for at least two other components are still loading, including:

- [ ] The pages render correctly, are accessible, and are responsive.

2. If components have been modified, VRTs have been run on this branch:

- [ ] VRTs have been run and looked at.
- [ ] Any VRT changes have been accepted (by reviewer and/or PR author), or there are no changes.

## To-do list

- [ ] I have read the [contribution guidelines](/.github/CONTRIBUTING.md).
- [ ] I have updated relevant storybook stories and templates.
- [ ] I have tested these changes in Windows High Contrast mode.
- [ ] If my change impacts **other components**, I have tested to make sure they don't break.
- [ ] If my change impacts **documentation**, I have updated the documentation accordingly.
- [ ] ✨ This pull request is ready to merge. ✨

### CODE REVIEW

**Comment by jenndiaz** [comment]
mod file is currently not generating mods of opacity checkerboard, this is being addressed in #2322

File: components/colorslider/index.css [hunk 1]
```diff
@@ -105,7 +120,7 @@
 
 /* Visually hidden "range" input field */
 .spectrum-ColorSlider-slider {
-	opacity: 0;
+	opacity: 0%;
```
Comment (jenndiaz) [inline components/colorslider/index.css:123]: style lint wants opacities in %

**Comment by pfulton** [comment]
@Westbrook I'd be curious to get your thoughts on this refactor. I know this was implemented differently in SWC (not a component) vs. how we have it in our repo (a "component"). What level of churn (if any) would this cause for you all?

**Review by Westbrook — COMMENTED** [review]
I'm not sure the right path for continued consumption of this approach in SWC without seeing the actual published code. For us to ship it as we have it needs to be staic _somewhere_ in the output, separate from the consuming patterns.

Things that I do know:
- using CSS preprocessing means that there are _multiple_ definitions of the CSS that can be shipped to users, rather than a single copy.
- we do have users that leverage the Opacity Checkerboard outside of the Spectrum elements
- the stand alone class was a useful way to support normalization across the consumption of this in both Spectrum patterns and Adobe apps and features at large.

**Comment by jenndiaz** [comment]
> I'm not sure the right path for continued consumption of this approach in SWC without seeing the actual published code. For us to ship it as we have it needs to be staic _somewhere_ in the output, separate from the consuming patterns.
> 
> Things that I do know:
> 
> * using CSS preprocessing means that there are _multiple_ definitions of the CSS that can be shipped to users, rather than a single copy.
> * we do have users that leverage the Opacity Checkerboard outside of the Spectrum elements
> * the stand alone class was a useful way to support normalization across the consumption of this in both Spectrum patterns and Adobe apps and features at large.

@pfulton and @Westbrook 

I think based on the fact that people are using this outside of Spectrum elements steers me towards a more simple approach. I'll look at simplifying the CSS if I can and documenting inclusion order more clearly.

I think this could be a good option to be more of "utility class" like we are looking at with background layers.

**Pull request closed without merging**